### DISCUSSION TITLE
Struggling to fly with only flow deck installed (manually via cfclient)

### DISCUSSION THREAD

**Discussion by kenblu24** [bitcraze/discussions#1087] [Q&A]
Hello, I've successfully installed the latest cfclient to my Windows machine and am flying a crazyflie manually using an xbox controller through cfclient. It only has the flow deck installed, and it seems to struggle to take off, and applying any inputs to the quad to stabilize it seems to cause it to fall out of the air for a split second. When using the assist mode, the drone sometimes fails to take off. We also plan to use the AI camera in conjunction with the flow deck, but installing this seems to make it too heavy, and the drone can't seem to make it higher than the ground effect region.

I'm also struggling with cfclient's UI; I've had trouble getting cfclient to take input mappings. Is this related? It doesn't seem to allow for more than 79% throttle, and there appears to be a 30% deadzone which is quite large and makes taking off manually really difficult.

**Comment by knmcguire** [comment]
hi! That is a bit worrying. Is this a brand new crazyflie? Have you done the battery test in the cfclient's console tab?

**Reply by kenblu24** [reply]
whoops forgot to actually hit the reply button. Yes, and the battery seems fine. It hasn't been used before, but I'm not the one who assembled it (though I did go through and check the getting started guide). Propeller test also passes if I put it on a medium-soft mousepad.

I tried flying it in a different environment, and it no longer drops out of the sky. I think there may have been RF interference previously, because the connection kept dropping despite being close to the PA radio. I also tried re-updating the firmware on every crazyflie, even though I'd already done it before. All that done, I managed to get one perfect manual flight with one CF using a controller in cfclient with both the camera and optical flow decks installed; whatever power issue was ocurring before seems gone now. After that, I have no idea what changed, but each subsequent flight has been a disaster with the CF uncontrollably drifting in manual mode.

We've got a couple CFs to work with, so I'll try some of the other ones. I've tried balancing the battery, but it feels like there's else something in the hardware/software fighting me.

**Reply by knmcguire** [reply]
Let us know if it works better for you! it might also be that the battery connector has some kind of corrosive layer on it, which can disturb the flow of power. This can be solved by unplugging and replugging many times. It really seems like it is a power issue indeed.
